internal/clients/telegram/state/handler.go

```diff
@@ -0,0 +1,118 @@
+package state
+
+import (
+	"log"
+	"strings"
+
+	"github.com/m-kuzmin/daily-reporter/internal/clients/telegram/response"
+	"github.com/m-kuzmin/daily-reporter/internal/clients/telegram/update"
+	"github.com/m-kuzmin/daily-reporter/internal/util/option"
+)
+
+type Handler interface {
+	PrivateTextMessage(update.PrivateTextMessage) Transition
```

Comment:
Add contexts to these so that GitHub API calls can use the same context that is present in getUpdates and such.

Reply:
Moved to the project board for now.